OpenDocumentReader/CoreWrapper.mm

```diff
@@ -47,6 +47,26 @@ - (bool)translate:(NSString *)inputPath into:(NSString *)outputPath with:(NSStri
             auto inputPathC = [inputPath cStringUsingEncoding:NSUTF8StringEncoding];
             auto inputPathCpp = std::string(inputPathC);
             
+            odr::FileType fileType;
+            try {
+                const auto types = odr::OpenDocumentReader::types(inputPathCpp);
+                if (types.empty()) {
+                    _errorCode = @(-5);
+                    return false;
+                }
+
+                fileType = types.back();
+            } catch (odr::UnsupportedFileType &e) {
```

Comment:
I think this should never throw but it might be a good idea to protect against it